pkg/plugins/runtime/k8s/plugin.go

```diff
@@ -232,9 +232,9 @@ func addValidators(mgr kube_ctrl.Manager, rt core_runtime.Runtime, converter k8s
 	mgr.GetWebhookServer().Register("/validate-v1-service", &kube_webhook.Admission{Handler: &k8s_webhooks.ServiceValidator{}})
 	log.Info("Registering a validation webhook for v1/Service", "path", "/validate-v1-service")
 
-	client, ok := k8s_extensions.FromNonCachedClientContext(rt.Extensions())
+	client, ok := k8s_extensions.FromSecretClientContext(rt.Extensions())
```

Comment:
Is it possible to receive a request in the `SecretValidator` webhook which is not for `kuma-system` namespace?

Reply:
no

```
  - name: secret.validator.kuma-admission.kuma.io
    namespaceSelector:
      matchLabels:
        kuma.io/system-namespace: "true"
```